project/SensibleSettings.scala

```diff
@@ -13,7 +13,8 @@ object Sensible {
 
   lazy val settings = Seq(
     ivyLoggingLevel := UpdateLogging.Quiet,
-    conflictManager := ConflictManager.strict,
+    // FIXME: how to be less strict for typesafe config? slick and akka pull in different versions
+    //conflictManager := ConflictManager.strict,
```

Comment:
cool, yes that's intentional so we know when we have two things asking for different versions and then we can investigate further.